test/integration/commands/run/test_run_slurm_scenarios.py

```diff
@@ -486,3 +486,137 @@ def test_run_with_additional_files(
     else:
         exit_code = as_exp.autosubmit.run_experiment(expid=as_exp.expid)
         _assert_exit_code(final_status, exit_code)
+
+
+@pytest.mark.xdist_group("slurm")
+@pytest.mark.slurm
+@pytest.mark.parametrize("wrappers, run_type", [
+    (
+        {
+            "WRAPPERS": {
+                "MAX_WRAPPED": 2,
+                "WRAPPER": {"JOBS_IN_WRAPPER": "job", "TYPE": "horizontal"},
+                "SECOND_WRAPPER": {"JOBS_IN_WRAPPER": "other", "TYPE": "horizontal"},
+            }
+        },
+        "run",
+    ),
+    (
+        {
+            "WRAPPERS": {
+                "WRAPPER": {
+                    "JOBS_IN_WRAPPER": "job",
+                    "TYPE": "horizontal",
+                    "MAX_WRAPPED": 2,
+                },
+                "SECOND_WRAPPER": {
+                    "JOBS_IN_WRAPPER": "other",
+                    "TYPE": "horizontal",
+                    "MAX_WRAPPED": 2,
+                },
+            }
+        },
+        "run",
+    ),
+    (
+        {
+            "WRAPPERS": {
+                "MAX_WRAPPED": 2,
+                "WRAPPER": {"JOBS_IN_WRAPPER": "job", "TYPE": "horizontal"},
+                "SECOND_WRAPPER": {"JOBS_IN_WRAPPER": "other", "TYPE": "horizontal"},
+            }
+        },
+        "inspect",
+    ),
+    (
+        {
+            "WRAPPERS": {
+                "WRAPPER": {
+                    "JOBS_IN_WRAPPER": "job",
+                    "TYPE": "horizontal",
+                    "MAX_WRAPPED": 2,
+                },
+                "SECOND_WRAPPER": {
+                    "JOBS_IN_WRAPPER": "other",
+                    "TYPE": "horizontal",
+                    "MAX_WRAPPED": 2,
+                },
+            }
+        },
+        "inspect",
+    ),
+    (
+            {
+                "WRAPPERS": {
+                    "WRAPPER": {
+                        "JOBS_IN_WRAPPER": "job&other",
```

Comment:
👍 I remember somewhere we had an issue with & or spaces not working. Good to have more tests with these different syntax options!